docs/src/main/paradox/docs/getting-started/running-nexus/index.md

```diff
@@ -542,133 +501,9 @@ $ kubectl get configmap/config -o yaml | grep public.ip:
-```
-
-Example
-:
-```
-$ kubectl apply -f $MINI/iam.yaml
-service/iam created
-service/iam-hd created
-statefulset.apps/iam created
-ingress.extensions/iam created
-ingress.extensions/iam-direct created
-$ kubectl wait pod iam-0 --for condition=ready --timeout=180s
-pod/iam-0 condition met
-$ curl -s "http://$NEXUS/iam" | jq
-{
-  "name": "iam",
-  "version": "1.1.0"
-}
-$ curl -s "http://$NEXUS/v1/acls/" | jq
-{
-  "@context": [
-    "https://bluebrain.github.io/nexus/contexts/resource.json",
-    "https://bluebrain.github.io/nexus/contexts/iam.json",
-    "https://bluebrain.github.io/nexus/contexts/search.json"
-  ],
-  "_total": 1,
-  "_results": [
-    {
-      "@id": "http://192.168.39.171/v1/acls",
-      "@type": "AccessControlList",
-      "acl": [
-        {
-          "identity": {
-            "@id": "http://192.168.39.171/v1/anonymous",
-            "@type": "Anonymous"
-          },
-          "permissions": [
-            "schemas/write",
-            "views/write",
-            "files/write",
-            "permissions/write",
-            "acls/write",
-            "realms/write",
-            "projects/read",
-            "acls/read",
-            "organizations/create",
-            "organizations/write",
-            "resources/write",
-            "realms/read",
-            "projects/create",
-            "permissions/read",
-            "resources/read",
-            "organizations/read",
-            "resolvers/write",
-            "events/read",
-            "views/query",
-            "projects/write"
-          ]
-        }
-      ],
-      "_path": "/",
-      "_rev": 0,
-      "_createdAt": "1970-01-01T00:00:00Z",
-      "_createdBy": "http://192.168.39.171/v1/anonymous",
-      "_updatedAt": "1970-01-01T00:00:00Z",
-      "_updatedBy": "http://192.168.39.171/v1/anonymous"
-    }
-  ]
-}
-$
-```
-
-#### Deploy Admin
-
-Admin is the service that manages organizations, projects and their configuration.
-
-Command
-:
-```
-kubectl apply -f $MINI/admin.yaml && \
-  kubectl wait pod admin-0 --for condition=ready --timeout=180s
-```
-
-Example
-:
-```
-$ kubectl apply -f $MINI/admin.yaml
-service/admin created
-service/admin-hd created
-statefulset.apps/admin created
-ingress.extensions/admin created
-ingress.extensions/admin-direct created
-$ kubectl wait pod admin-0 --for condition=ready --timeout=180s
-pod/admin-0 condition met
-$ curl -s "http://$NEXUS/admin" | jq
-{
-  "name": "admin",
-  "version": "1.1.0"
-}
-$ curl -s "http://$NEXUS/v1/projects" | jq
-{
-  "@context": [
-    "https://bluebrain.github.io/nexus/contexts/admin.json",
-    "https://bluebrain.github.io/nexus/contexts/resource.json",
-    "https://bluebrain.github.io/nexus/contexts/search.json"
-  ],
-  "_total": 0,
-  "_results": []
-}
-$
-```
-
-#### Deploy KG
+#### Deploy Delta
 
-KG is the service that manages user defined resources, their schemas and configuration like resolvers, views etc.
+Delta is the service providing Nexus REST API/
```

Comment:
Should there be a slash at the end of this sentence + "the" is missing? 🤔
E.g. "Delta is the service providing the Nexus REST API."

Reply:
Fixed.